http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -771,7 +734,7 @@ private static void closeQuietly(LifeCycle service)
                 service.stop();
             }
         }
-        catch (InterruptedException ignored) {
+        catch (InterruptedException ie) {
```

Comment:
We can name this `e`, as we do elsewhere. I thought it needed to be `ignored` to prevent a warning, but apparently that's only a problem if the catch block is empty.